frontend/src/modules/core/components/table/AppTable.tsx

```diff
@@ -26,6 +26,7 @@ type Props<T> = {
   initialSortingState?: SortingState;
   getRowId?: (originalRow: T, index: number, parent?: Row<T>) => string;
   variant?: 'form' | 'table';
+  readOnly?: boolean;
```

Comment:
probably `disabled` would make more sense given that it is used more often in this project